readme.md

```diff
@@ -135,23 +135,23 @@ $ npm -v
 
 * **Question:** What is `path` and where does it come from?
 
-* **Your Answer:**
+* **Your Answer:** It's the path of the file that calls path. It comes from the node module.
```

Comment:
`path` itself _is_ the node module. `path` has different methods attached to it which can produce the file path.